blogs/posts/2024-01-16_nearest_neighbour.qmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "Nearest neighbour imputation"
+author: "Jacqueline Grout with Tom Jemmett"
+date: "2024-01-16"
+categories: [learning]
+---
+
+Recently I have been gathering data by GP practice, from a variety of different sources. The ultimate purpose of my project is to be able to report at an ICB/sub-ICB level. The various datasets cover different timescales and consequently changes in GP practices over time have left me with mismatching datasets.
+
+My approach has been to take as the basis of my project a recent GP List. Later in my project I will perform calculations at a GP practice level based on underlying need and the data for this need is a prevalence value from a dataset which is around 8 years old, and for which there is no update or alternative. From my recent list of 6454 practices, when I match to the need data, I am left with 151 practices without a need. If I remove these practices from the analysis then this could impact the analysis by sub-icb since often a group of practices in the same area could be subject to changes, mergers and reorganisation.
```

Comment:
Does 'need' need a short definition? I'm not sure what 'need' data is, exactly.

Reply:
I've tweaked the openeing section to try to make this an easier read and to aid understanding that need is in fact health need and include the example of what this is in my particular project